webrtc.html

```diff
@@ -6070,18 +6056,104 @@ <h2>Dictionary <code><a>RTCRtpParameters</a></code> Members</h2>
+            <dt><dfn data-idl><code>transactionId</code></dfn> of type <span class=
+            "idlMemberType"><a>DOMString</a></span>, required</dt>
+            <dd>
+              <p>An unique identifier for the last set of parameters applied.
+              Ensures that setParameters can only be called based on a previous
+              getParameters, and that there are no intervening changes.
+              <a>Read-only parameter</a>.</p></dd>
+            <dt><dfn data-idl><code>encodings</code></dfn> of type <span class=
+            "idlMemberType">sequence&lt;<a>RTCRtpEncodingParameters</a>&gt;</span>,
+            required</dt>
+            <dd>
+              <p>A sequence containing parameters for RTP encodings of
+              media.</p>
+            </dd>
             <dt><dfn data-idl><code>degradationPreference</code></dfn> of type
-            <span class="idlMemberType"><a>RTCDegradationPreference</a></span></dt>
+            <span class="idlMemberType"><a>RTCDegradationPreference</a></span>,
+            defaulting to <code>"balanced"</code></dt>
             <dd>
               <p>When bandwidth is constrained and the
               <code><a>RtpSender</a></code> needs to choose between degrading
               resolution or degrading framerate,
               <code>degradationPreference</code> indicates which is
-              preferred. If unset, the <code><a>RtpSender</a></code> defaults to
-              the <code>balanced</code> policy.</p>
-              <p>For an <code><a>RtpReceiver</a></code>,
-              <code>degradationPreference</code> is inapplicable and will
-              always be <code>undefined</code>.</p>
+              preferred.</p>
+            </dd>
+          </dl>
+        </section>
+      </div>
+      </section>
+     <section id="rtcreceivertpparameters">
+     <h3><dfn>RTCReceiveRtpParameters</dfn> Dictionary</h3>
+      <div>
+        <pre class="idl">dictionary RTCReceiveRtpParameters : RTCRtpParameters {
+             required sequence&lt;RTCRtpDecodingParameters&gt;  encodings;
+};</pre>
+        <section>
+          <h2>Dictionary <code><a>RTCReceiveRtpParameters</a></code> Members</h2>
+          <dl data-link-for="RTCReceiveRtpParameters" data-dfn-for="RTCReceiveRtpParameters"
+          class="dictionary-members">
+            <dt><dfn data-idl><code>encodings</code></dfn> of type <span class=
+            "idlMemberType">sequence&lt;<a>RTCRtpDecodingParameters</a>&gt;</span>,
+            required</dt>
+            <dd>
+              <p>A sequence containing information about incoming RTP encodings
+              of media.</p>
+            </dd>
+          </dl>
+        </section>
+      </div>
+      </section>
+      <section id="rtcrtpdecodingparameters">
+      <h3><dfn>RTCRtpDecodingParameters</dfn> Dictionary</h3>
+      <div>
+        <pre class="idl">dictionary RTCRtpDecodingParameters {
+             boolean             active = true;
```

Comment:
We don't need an `active` field for the receiver since it has no `setParameters`. Note the definition from ORTC:

> For an RTCRtpReceiver, indicates that this encoding is being decoded. Setting it to false causes this encoding to no longer be decoded. Setting it to true causes this encoding to be decoded.

So, this is meant to be something you can set to false to turn off decoding (saving resources I guess). Without the ability to set it to false it has no use.

Reply:
OK so there's no other way for it to be inactive? I'll remove it.

Reply:
@taylor-b When a sender sets `active = false`, what happens on the receiver's end?

Reply:
By "the receiver", do you mean an SFU receiving the simulcasted streams, or an `RTCRtpReceiver` on the other end receiving a continuous stream from the SFU?

If only one encoding is deactivated, I'd expect the SFU to switch to forwarding another encoding, and the downstream endpoints may notice a decrease in quality. If all encodings are deactivated, I guess eventually an RTP timeout would occur and the track `muted` state would be set, as described [here](https://www.w3.org/TR/webrtc/#mediastreamtrack).

Reply:
OK removing it.